mlir/test/Conversion/TosaToRock/reductions/tosa-to-rock-reduce-e2e.mlir

```diff
@@ -1,9 +0,0 @@
-// RUN: rocmlir-opt --tosa-partition="anchor-ops=tosa.reduce_sum" --xmodel-async-graph --xmodel-target-kernels="targets=%arch" %s | rocmlir-driver -host-pipeline highlevel | rocmlir-gen -ph -print-results -rand 1 -rand_type float -fut test_basic --verifier clone - | rocmlir-driver -host-pipeline xmodel -kernel-pipeline full -targets %arch | xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CLONE
-// cat %s | rocmlir-driver -host-pipeline xmodel -kernel-pipeline full -targets %arch | xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CLONE
-// CLONE: [1 1 1]
-// CLONE-NEXT: Unranked Memref base
-
-func.func @test_basic(%arg0: tensor<200x100xf32>) -> tensor<200x1xf32> {
-  %1 = "tosa.reduce_sum"(%arg0) {axis = 1 : i64} : (tensor<200x100xf32>) -> tensor<200x1xf32>
```

Comment:
Im removing this for now. It has become tricky do hookup to tosa to get standalone reductions to work in the absence of copy opt as a separate pass. We can revisit them, if go on to optimize the standalone kernel -- i'd think we would want to have the copy opt seperate pass run after align tiling.

For now, we are good for fusions with gemm-like ops.